src/app/components/header.tsx

```diff
@@ -62,7 +63,12 @@ export const Header: React.FC<HeaderProps> = memo(props => {
     >
       {onClose ? (
         <Box flexBasis="20%">
-          <IconButton alignSelf="center" icon={FiArrowLeft} iconSize="16px" onClick={onClose} />
+          <IconButton
+            alignSelf="center"
+            icon={closeIcon ? FiX : FiArrowLeft}
+            iconSize="16px"
+            onClick={onClose}
+          />
         </Box>
```

Comment:
I see this isn't your code, but bad accessibility. click handler should be on outer Box (which should be a button)

Reply:
exactly, thanks